docs/SYNC_MODE.md

```diff
@@ -84,16 +76,14 @@ with Transformer() as transformer:
             )
         )
 ```
+(see [tap-trello sync.py](https://github.com/singer-io/tap-trello/blob/master/tap_trello/sync.py) for an example)
 
 
 ## How to handle child streams
 If there are child streams, this means that they rely on some piece of information from a corresponding parent.
-if child stream `CHILD` relies on parent stream `PARENT`'s id, then if we sync `CHILD` without `PARENT` being selected, the python process will error out with an unclear error message.
-The tap can do one of two things to handle this:
-- grab parent ids in child stream sync function:
-  - in the function to sync `CHILD`, we can grab all id's for and iterate through that list. This will allow us to sync `CHILD` without selecting `PARENT`
-- log a clear error that parent stream `PARENT` must be selected if we want to sync `CHILD`
-  - make it clear in the logs: "in order to sync `CHILD`, you must also select `PARENT`
+To handle this, the tap must grab parent ids in child stream sync function
+(see [tap-trello streams.py](https://github.com/singer-io/tap-trello/blob/master/tap_trello/streams.py) in the ClientStream sync method)
```

Comment:
For these examples, I don't think Trello is a standard we should promote (due to the unique Mixin pattern that's in there), for teaching, I've been using `tap-adroll` for its simplicity. The `Ads` stream is probably a good, clean example: https://github.com/singer-io/tap-adroll/blob/138fc92dc4fb17c4b9446a3cf998b34b288b3e4a/tap_adroll/streams.py#L55

These links should also probably be at a specific SHA, in case anything changes in the future.